File: Workshop 2/The Happy Pirate Java/src/View/EngMenu.java
package View;

import java.io.Console;

/**
 * Created by henry on 2016-09-20.
 */
public class EngMenu extends View.Console implements iMenu {


    public EngMenu() {
        super();
    }

    @Override
    public boolean AuthenticateField() {
        return GetAns("1: Login\n2: Register\nAns: ").equals("1");
    }

    @Override
    public String[] LoginField() {
        String username = GetAns("Username: ");
        String password = GetAns("Password: ");

        String[] arr = {username, password};
        return arr;
    }

    @Override
    public void RegistrationField() {

    }

    @Override
    public void MemberMenu() {

    }

    @Override
    public void SecretaryMenu() {

    }

    @Override
    public void TreasurerMenu() {

    }

    @Override
    public void BoatMenu() {

    }

}

File: Workshop 2/The Happy Pirate C#/YatchClub/YatchClub/View/EngMenu.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using YatchClub.Model;
using YatchClub.Users;

namespace YatchClub.View
{
    class EngMenu : iMenu
    {
        // login and registration fields
        public string AuthenticateField() {
            return "=== Authentication ===\n1. Login\n2. Register\n?: ";
        }
        public string RegistrationField() {
            return "=== Register Menu ===\n1. Username\n2. Email\n3. Password\n";
        }
        public string[] LoginField() {
            Console.WriteLine("=== Login Menu ===");
            return new string[] { "Username", "Password" };
        }



        public string MemberField(Member member)
        {
            throw new NotImplementedException();
        }

        public string SecretaryField(Secretary secretary)
        {
            throw new NotImplementedException();
        }

        public string TreasurerField(Treasurer treasurer)
        {
            throw new NotImplementedException();
        }

        public string BoatField(Boat boat)
        {
            throw new NotImplementedException();
        }
    }
}
